lib/galaxy/managers/base.py

```diff
@@ -1204,7 +1206,30 @@ def parse_date(self, date_string):
         if match:
             date_string = ' '.join(group for group in match.groups() if group)
             return date_string
-        raise ValueError('datetime strings must be in the ISO 8601 format and in the UTC')
+        raise ValueError("datetime strings must be in the ISO 8601 format and in the UTC")
+
+    def create_str_orm_filter(self, attr, op, val, sep=","):
+        """
+        Create an ORM filter function for a string column/property to support
+        values that need to be parsed differently depending on the operation,
+        e.g. the `in` operation requires splitting on `,` while for `eq` and
+        `like` no parsing is needed.
+        """
+        def _create_str_orm_filter(model_class=None):
+            column = self.model_class.table.columns.get(attr)
+            if column is None:
+                # could be a property (hybrid_property, etc.) - assume we can make a filter from it
+                column = getattr(self.model_class, attr)
+            if op not in ["eq", "in", "like"]:
+                return None
+            converted_op = self._convert_op_string_to_fn(column, op)
+            if op == "in":
+                converted_val = val.split(sep)
```

Comment:
I think that should ideally be handled in the parameter model, it's an artifact of the q/qv syntax, but down in the manager we should work with lists ?

Reply:
Agree, that would be certainly ideal :)

Reply:
Given how the q/qv syntax handling is currently implemented, I think that'd be a whole rewrite, at which point we'd better get rid of the q/qv syntax. So, I don't think it can be 22.01 bugfix material.

Reply:
No worries if 22.01 as a target doesn't work. Can certainly find a workaround until the fix becomes available.